README.md

```diff
@@ -2,4 +2,4 @@
 
 [![Build Status](https://travis-ci.org/code-ci/bacaroso.svg)](https://travis-ci.org/code-ci/bacaroso)
 [![Code Climate](https://codeclimate.com/github/code-ci/bacaroso/badges/gpa.svg)](https://codeclimate.com/github/code-ci/bacaroso)
-[![Test Coverage](https://codeclimate.com/github/code-ci/bacaroso/badges/coverage.svg)](https://codeclimate.com/github/code-ci/bacaroso/coverage)
+[![Coverage Status](https://coveralls.io/repos/code-ci/bacaroso/badge.svg?branch=task%2Fcreate_model_country&service=github)](https://coveralls.io/github/code-ci/bacaroso?branch=task%2Fcreate_model_country)
```

Comment:
@RafaelRNunes Please change it back to master branch, it isn't necessary to change it to a specific branch.